php/pantheon/checks/database.php

```diff
@@ -55,7 +55,7 @@ protected function countRows() {
 			$this->alerts[] = array('code'=>0, 'message'=> sprintf("Found %s rows in the options table.", $options_table->TABLE_ROWS), 'class'=>'ok');
 		} 
 
-		$autoloads = $wpdb->get_results("SELECT * FROM wp_options WHERE autoload = 'yes'");
+		$autoloads = $wpdb->get_results("SELECT * FROM " . $wpdb->prefix . "options WHERE autoload = 'yes'");
```

Comment:
To ensure this works as expected on WordPress Multisite, can you instead use `{$wpdb->options}` ?